app.js

```diff
@@ -46,24 +54,73 @@ var myColor = 'NO';
 
 if (color.toUpperCase() === myColor) {
   alert('Correct! I love green.');
+  count++;
 } else {
   alert('Nice try, but green is actually my favorite color.  I love being surrounded by plants!');
 }
 
-// var myMovement = ['yoga', 'hiking', 'qi gong', 'dancing'];
-// var movement = prompt('Name one of my favorite forms of exercise.');
-//
-// if (myMovement.indexOf(movement)!== -1) {
-//   alert('You\'re right, ' + movement + ' is one of my favorite forms of exercise.');
+var myMovement = ['yoga', 'hiking', 'qi gong', 'dancing'];
+var movement = prompt('Name one of my favorite forms of exercise.');
+
+if (myMovement.indexOf(movement)!== -1) {
+  alert('You\'re right, ' + movement + ' is one of my favorite forms of exercise.');
+  count++;
+} else {
+  alert('That\'s not one of my favorites. I prefer yoga, qi goong, hiking, and dancing.');
+}
+
+
+var beverage = prompt('What is one of my favorite beverages?');
+
+if(beverage.toLowerCase === 'kombucha' || 'green smoothies' || 'coconut water' || 'herbal tea'){
+  console.log('You guessed correct! My favorite beverages are kombucha, green smoothies, coconut water, and herbal tea.');
+  count++;
+} else {
+  console.log('That is incorrect.  We must have different taste in drinks.');
+}
+
+
+
+
+function checkGuess() {
+
+    var yearsVegan = 18;
+    var counter = 0;
+    var correct = false;
+
+  while ((counter > 4) && (correct === false)) {
+  var guess = prompt('Can you guess how many years I have been vegan?');
```

Comment:
Good naming here to call the variable 'guess'. Explicit variable naming will save you so many headaches later!